### DISCUSSION TITLE
it/aire/

### DISCUSSION THREAD

**Discussion by giscus** [matteocontrini/blog#11] [Comments]
# it/aire/

Un eroe dall’estero mi racconta la sua avventura per l’iscrizione all’AIRE, l’Anagrafe italiani residenti all’estero. Sei pronto per affrontare la tabella dei procedimenti ad istanza di parte?

https://matteosonoio.it/it/aire/

**Comment by belial90** [comment]
Aggiungo la mia esperienza:
Dopo esser riuscito a fare la domanda di iscrizione all’AIRE, dopo averci provato un paio di volte e aver ricevuto indicazione che una PEC con la pratica fosse stata inviata al mio comune di (ex)residenza, ho iniziato ad aspettare settimane. 
Senza nessuna conferma sulla pratica da parte del comune, ho dovuto mandare un familiare agli uffici a chiedere aggiornamenti sulla pratica da parte mia. Ovviamente i dipendenti del comune non avevano MAI aperto la casella pec, a detta loro, e hanno infine concluso l’iscrizione. 😩
